src/features/writing-sum-form/lib.ts

```diff
@@ -0,0 +1,4 @@
+export const convertToNumber = (value: string): number => {
+    return Number(value.replace(/[^0-9.,]/g, ''))
+}
```

Comment:
Сделай проверку на NaN
```suggestion
export const convertToNumber = (value: string): number => {
    const result = Number(value.replace(/[^0-9.,]/g, ''))
    
    if (isNan(result)) {
      return 0; // или что ты хочешь вернуть если nan
    }
    
    return result
}
```